### ISSUE TITLE
Login / Sign Up Page Conversion

### ISSUE THREAD

**Issue by jobenjada** [formbricks/design#14]
**Problem**

We have a 66% drop off rate on the signup / login page:

![grafik](https://github.com/formbricks/design/assets/72809645/201a4fdf-ad54-4ac5-9346-57cd0eeb724c)

**Solution**
Optimize conversion following best practices.

I came across this Sign Up Page:
![grafik](https://github.com/formbricks/design/assets/72809645/614a3b79-184d-47f0-96bf-fb45d278a384)

I like it a lot because:

1.  It reassures the value of the product
2.  It makes it credible with the image (some form of social proof)
3.  It does not have any input fields which elict "sign up angsts"
4.  It uses "Continue with..." wording to fight Sign Up Angst
5.  Offers Google SSO

**Next Actions**
I'll do some more research and make a mock up on Pen Pot to discuss here.

**Comment by jobenjada** [comment]
First version: https://github.com/formbricks/formbricks/pull/288

**Comment by Shubhdeep12** [comment]
How about having a gradient of green and white color in background? @jobenjada 

And, lets consider mobile screens also.

**Comment by jobenjada** [comment]
Thanks!!

I just opened a PR for the quick fixes and things I agree on.

Re wording: I'd like to keep "Log in" as well as "Continue..." because the "Log in" is an indicator that this is the login page. But when people without an account dont read that and click on "Continue with GitHub" we check if there is an account and if not we create one. So the SSO Buttons work for both cases on both screens (login & signup)

For the overpowering, I couldnt find a quick fix.

**Comment by sirkotsky** [comment]
@jobenjada I quickly put some ideas together to give you a better sense of what I mean (you can check the Penpot file, I left the artboard there):

0. Using a different background colour (something subtle) to bring the user's attention to a brighter block, and allocating more space to the form to avoid overpowering the actionable section with content;
1. Cleaning the spacing and (**very important**) avoid using pure black (#000)! Let's be gentle on our user's eyes!
2. The testimony might be a good and important block, but it doesn't have to drive all of the attention to itself. Still, let it breathe, give it some space, and enlarge it a bit. Consistent spacing, dim colours;
3. Use a proper logo. We can use the F instead of a full logotype, but we don't need to encircle it or anything;
4. Clean up the typography and the copy: if you separate login and registration, let's make it clear for the user what the buttons do (in this case, provide them with log in options); 
5. Same with the registration link: give it some space, avoid making it look like a text block, and make it clear what it does. 

![image](https://github.com/formbricks/design/assets/22578094/848edd32-59f8-49ac-97cd-4f505be4659a)

Either way, I'm merely trying to do 2 things here: bring the attention to the form and avoid confusing the user, but I'm sure there's a lot we can improve further. Let me know what you think!

**Comment by jobenjada** [comment]
very good suggestions, will work on it this week :)

**Comment by jobenjada** [comment]
Hey @sirkotsky !

Here is what I've built based on your suggestions:

![grafik](https://github.com/formbricks/design/assets/72809645/35b9f43a-3d71-4241-be71-e7ec065f4d28)

0. done, good idea
1. Where did you find #000 ? I tried to use Tailwind slate-900 mostly
2. Dimmed the colors and gave it some more padding, I totally see your point but this was my pareto solution :)
3. Done
4. Copy: I might be drawing too much from my own experience but I didnt read the all of the text on the example (GitStart), I just saw the GitHub logo and clicked. Thats what I liked so much about this page, it sucked me in. However, since the form behind "Email" wont work if people try to create an account on the login page, I updated it accordingly. On the sign up page we mostly funnel people who dont have an account yet and I like to keep it with "Continue"
5. Done
